app/pods/components/nav-bar/template.hbs

```diff
@@ -3,8 +3,8 @@
     <span class="hamburger-parent" onclick={{action 'toggleSidebar'}}>
             <i class="hamburger fas fa-bars"></i>
           </span>
-    <span>
-      {{#link-to 'index' activeClass=''}}
+    <span class="logo-parent">
+      {{#link-to 'index' activeClass='logo-link'}}
```

Comment:
Why is this required here?

Reply:
I've updated the code and remove these extra classes.